src/main/scala/beam/router/r5/NetworkCoordinator.scala

```diff
@@ -101,7 +101,6 @@ trait NetworkCoordinator extends LazyLogging {
         // Needed because R5 closes DB on write
         transportNetwork = KryoNetworkSerializer.read(path.toFile)
       }
-      .get
```

Comment:
Rather than removing, should we check and log/handle the failure scenario?

Reply:
use try/catch for handle/log error